docs/data-sources/compliance_framework.md

```diff
@@ -0,0 +1,75 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "mondoo_compliance_framework Data Source - terraform-provider-mondoo"
+subcategory: ""
+description: |-
+  Data source to return compliance frameworks in a Space
+---
+
+# mondoo_compliance_framework (Data Source)
+
+Data source to return compliance frameworks in a Space
+
+## Example Usage
+
+```terraform
+data "mondoo_compliance_framework" "frameworks" {
+  space_id = "your-space-1234567"
+  # space_mrn    = "your-space-mrn" 
```

Comment:
i think we should make this example more simple and not add comments

Reply:
To align the wording better with the `mondoo_assets` resource, I suggest we change the resource to:

```
data "mondoo_frameworks" "my_space" {
  space_id = "your-space-1234567"
}

output "framework_mrn" {
  value       = [for framework in data.mondoo_frameworks.my_space.frameworks : framework.mrn]
  description = "The MRN of the frameworks in the space."
}
```

> Note: we need to align `mondoo_policy` to also use the plural

Reply:
https://github.com/mondoohq/terraform-provider-mondoo/pull/119#discussion_r1680435705